tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/metrics/AllMetricsTest.java

```diff
@@ -45,7 +45,8 @@ public static WebArchive deploy() {
         WebArchive war = ShrinkWrap.create(WebArchive.class, "ftMetricAll.war")
                 .addClasses(AllMetricsBean.class)
                 .addPackage(Packages.UTILS)
-                .addPackage(Packages.METRIC_UTILS);
+                .addPackage(Packages.METRIC_UTILS)
+                .addAsManifestResource(EmptyAsset.INSTANCE, "beans.xml");
```

Comment:
Note that in a war, the `beans.xml` file must be named `WEB-INF/beans.xml` or `WEB-INF/classes/META-INF/beans.xml`. I'm not sure where does shrinkwrap add the resource when using `WebArchive#addAsManifestResource`...